clientinfoPlugin.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DNWS
+{
+  class clientinfoPlugin : IPlugin
+  {
+    protected static Dictionary<String, int> clientinfoDictionary = null;
+    public clientinfoPlugin()
+    {
+      if (clientinfoDictionary == null)
+      {
+        clientinfoDictionary = new Dictionary<String, int>();
+
+      }
+    }
+
+    public void PreProcessing(HTTPRequest request)
+    {
+      if (clientinfoDictionary.ContainsKey(request.Url))
+      {
+        clientinfoDictionary[request.Url] = (int)clientinfoDictionary[request.Url] + 1;
+      }
+      else
+      {
+        clientinfoDictionary[request.Url] = 1;
+      }
+    }
+    public HTTPResponse GetResponse(HTTPRequest request)
+    {
+      HTTPResponse response = null;
+      StringBuilder sb = new StringBuilder();
+      sb.Append("<html><body><h1></h1>");
+      // foreach (KeyValuePair<String, int> entry in clientinfoDictionary)
+      // {
```

Comment:
Please clean your code.

Reply:
It's not good to remove code by commenting it out, it's still there. You should remove them completely. We have git to track the changes, so you don't need to worry that you will lose the code.